getting-started/background-processing/TranslateWorker/TranslateWorker.csproj

```diff
@@ -1,15 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Firestore" Version="2.5.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
-    <PackageReference Include="Google.Cloud.Translation.V2" Version="2.1.0" />
+    <PackageReference Include="Google.Cloud.Translation.V2" Version="3.0.0" />

```

Comment:
Was the previous Firestore reference just unused?

Reply:
Nope, but it gets it transitively from the Transalation project so it did't need to be explicit.